soh/src/code/z_parameter.c

```diff
@@ -5949,6 +5950,107 @@ void Interface_Draw(PlayState* play) {
     CLOSE_DISPS(play->state.gfxCtx);
 }
 
+void Interface_DrawTotalGameplayTimer(PlayState* play) {
+    // Draw timer based on the Gameplay Stats total time.
+
+    if (CVarGetInteger("gGameplayStats.ShowIngameTimer", 0) && gSaveContext.fileNum >= 0 && gSaveContext.fileNum <= 2) {
+
+        s32 X_Margins_Timer = 0;
+        if (CVarGetInteger("gIGTUseMargins", 0) != 0) {
+            if (CVarGetInteger("gIGTPosType", 0) == 0) {
+                X_Margins_Timer = Left_HUD_Margin;
+            };
+        }
+        s32 rectLeftOri = OTRGetRectDimensionFromLeftEdge(24 + X_Margins_Timer);
```

Comment:
I'm not super familiar with our margin/HUD placement system but I feel like a lot of this code should be the same and re-used for all elements. Nothing to do about this now just a note.

Reply:
Yeah I've essentially copied these from some other HUD element that got the placements from the Cosmetics editor. It'd be nice to overhaul all of them at some point because frankly, they're a mess.

Reply:
Made an issue for it. https://github.com/HarbourMasters/Shipwright/issues/2914